### ISSUE TITLE
Could not authorize you from GitHub because of "Invalid credentials".

### ISSUE THREAD

**Issue by bluet** [codetriage/CodeTriage#1671]
## Expected Behavior

Click "Log in" or "Sign Up" button should log me in.

## Current Behavior

Shows `Could not authorize you from GitHub because of "Invalid credentials".` on page top.

## Possible Solution

None.

## Steps to Reproduce (for bugs)

1. At https://www.codetriage.com/
2. Click "Log in" at top-right.
3. After some redirection and refresh, it shows error.

## Context

Need to log in but unable to.

## Your Environment

* Version used:
* Browser Name and version: Google Chrome  104.0.5112.101 64bit
* Operating System and version (desktop or mobile): Ubuntu 22.04 Desktop
* Link to your project:

**Comment by schneems** [comment]
Were you able to get in or still  not working?

**Comment by bluet** [comment]
Hello, yes I'm still experiening this error.

![螢幕快照 2022-11-04 17-42-06](https://user-images.githubusercontent.com/51141/199942297-d73700bd-4593-4d69-a232-436903e9c4fe.png)

**Comment by schneems** [comment]
are you able to authorize other GitHub applications? For instance could you log into circleCI with your GitHub account?

If so, then I think the next step might be contacting GitHub support as it seems everything is okay on our end (others are able to sign up still). I’m wondering if GitHub employees could see if there are some exceptions logged against your oauth request on their end.

**Comment by bluet** [comment]
@schneems yes I can use github to login/register other applications/services without problem. The error only happens on CodeTriage.
